daemon/openrazer_daemon/dbus_services/dbus_methods/argb_controller.py

```diff
@@ -0,0 +1,317 @@
+from openrazer_daemon.dbus_services import endpoint
+
+
+def _get_channel_brightness(self, channel):
+    driver_path = self.get_driver_path(channel + '_led_brightness')
+    with open(driver_path, 'r') as driver_file:
+        return float(driver_file.read().strip()) / (255.0 / 100.0)
+
+
+@endpoint('razer.device.lighting.channel1', 'getChannel1Brightness', out_sig='d')
+def get_channel1_brightness(self):
```

Comment:
I really dislike having all these methods copy-pasted 6 times but I guess it fits into the current API design...

Reply:
@brett-smith Let's deviate here a bit from the current API design and just make a `getChannelBrightness` with parameter for channel ID. We also need a "nr. of channels" parameter somewhere that can be used. But I really really don't want to have 6 copies of these methods (even if it makes using this part of the API a bit more annoying as it's different)